client/src/components/shared/Info/Notices.js

```diff
@@ -11,14 +12,20 @@ type NoticesProps = {|
 
 const Notices = ({data}: NoticesProps) => (
   <InfoButton
-    label="Zmluvy so štátom"
+    label="Obstarávania"
     count={data.count}
     priceSum={data.total_final_value_amount_eur_sum}
     list={data.most_recent}
     buildItem={(notice: NoticeNew) => (
       <li key={notice.id} title={notice.title}>
-        {`${notice.client_name}, `}
-        <ShowNumberCurrency num={notice.total_final_value_amount} />
+        <ExternalLink url={`${window.location.host}/obstaravania/${notice.notice_id}`}>
+          <div>
+            <b>{notice.client_name}</b>
+            {notice.total_final_value_amount ? ', ' : ''}
+            <ShowNumberCurrency num={notice.total_final_value_amount} />
+          </div>
```

Comment:
Empty space in this div is clickable as part of the link when title wraps. Not a huge deal, just kind of weird.

Reply:
Yup, gonna change that.